.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Expected behavior
+A clear and concise description of what you expected to happen.
+
+### To Reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected causes and correction methods
+Creating helps someone modify, even if you're unsure
```

Comment:
What does this mean?

Reply:
Umm.. It meant that even if you don't know the exact reason, please write it down.
I will update it clearly.